pkg/inngest/function.go

```diff
@@ -369,7 +369,7 @@ func (f Function) Validate(ctx context.Context) error {
 	// Validate cancellation expressions
 	for _, c := range f.Cancel {
 		if c.If != nil {
-			if exprErr := expressions.Validate(ctx, *c.If); exprErr != nil {
+			if exprErr := expressions.Validate(ctx, expressions.DefaultRestrictiveValidationPolicy(), *c.If); exprErr != nil {
```

Comment:
refs: https://linear.app/inngest/issue/INN-4696/fail-app-syncs-containing-cel-macros-in-cancelon-expressions